build.py

```diff
@@ -65,21 +65,22 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.12.0dev':
-      ('21.07dev',   # triton container
-       '21.06',      # upstream container
-       '1.8.0',      # ORT
-       '2021.2.200', # ORT OpenVINO
-       '2021.2')     # Standalone OpenVINO
+    "2.12.0dev": (
+        "21.07dev",  # triton container
+        "21.06",  # upstream container
+        "1.8.0",  # ORT
+        "2021.2.200",  # ORT OpenVINO
+        "2021.2",
+    )  # Standalone OpenVINO
 }
 
-EXAMPLE_BACKENDS = ['identity', 'square', 'repeat']
-CORE_BACKENDS = ['tensorrt', 'ensemble']
+EXAMPLE_BACKENDS = ["identity", "square", "repeat"]
+CORE_BACKENDS = ["tensorrt", "ensemble"]
 NONCORE_BACKENDS = [
     'tensorflow1', 'tensorflow2', 'onnxruntime', 'python', 'dali', 'pytorch',
-    'openvino', 'fil', 'fastertransformer'
+    'openvino', 'fil', 'fastertransformer', 'tflite'
```

Comment:
I thought we had discussed that this should be "armnn" as we don't want to advertise and support a general tflite backend as part of this effort.